src/main/java/tic/tac/toe/ConsoleInputOutput.java

```diff
@@ -0,0 +1,20 @@
+package tic.tac.toe;
+import java.util.Scanner;
+
+public class ConsoleInputOutput implements InputOutput {
+    public int getInput() {
+        int returnValue = 0;
+        Scanner sc = new Scanner(System.in);
+        try {
+            returnValue = sc.nextInt();
+        }
+        catch (Exception e) {
+
+        }
+        return returnValue;
+    }
+
+    public void sendOutput(String output) {
+        System.out.println(output);
```

Comment:
change to print, not println, include new line with string

Reply:
My initial thought was being slightly surprised that a newline was "inserted" when calling `sendOutput`, however after thinking about it, I am curious at to what @elijahsullivan and @jchung722 think.